apm-agent-core/src/main/java/co/elastic/apm/agent/bci/IndyBootstrap.java

```diff
@@ -296,6 +315,10 @@ static void setJavaBaseModule(Class<?> targetClass) throws Throwable {
             .invoke(targetClass);
     }
 
+    public static void logExceptionThrownByAdvice(Throwable exception) {
+        LoggerFactory.getLogger(IndyBootstrap.class).error("Advice threw an exception, this should never happen!", exception);
```

Comment:
If there's no reason not to, please use a static `logger` field.

Reply:
I think we can't use a static logger field here for the same reason we can't use it in the `bootstrap()`-method.
However, i'll add a static `logger()` method to keep the comment about it in a single place.

Reply:
That's very true, thanks for reminding me to look at my own comment 🤦‍♂️